packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -51,6 +135,62 @@ - (void)hide {
   _mapView.hidden = YES;
 }
 
+#pragma mark - Implementations of JSON conversion functions.
```

Comment:
I think you want to move all the static methods to outside the `FLTGoogleMapController` implementation.

Reply:
done